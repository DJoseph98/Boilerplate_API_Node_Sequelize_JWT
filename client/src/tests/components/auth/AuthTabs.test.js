import React from 'react';
import { shallow } from 'enzyme';
import AuthTabs from '../../../components/auth/AuthTabs';

describe('AuthTabs component', () => {
    it('should render AuthTabs component', () => {
        const wrapper = shallow(<AuthTabs/>)
        expect(wrapper).toMatchSnapshot()
    })
})