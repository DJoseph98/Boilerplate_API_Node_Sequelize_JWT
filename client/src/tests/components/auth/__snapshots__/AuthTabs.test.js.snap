// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthTabs component should render AuthTabs component 1`] = `ShallowWrapper {}`;
